test/fuzz/gen.ml

```diff
@@ -67,137 +76,177 @@ let expr_available_2_i64_i32 (env : Env.t) =
     ]
   else []
 
-let expr_available_2_f32_i32 (env : Env.t) =
+let expr_available_2_f32_i32 (env : Env.t) : _ condgen =
+  return
+  @@
   if B.memory_exists env then [ pair B.f32_store (const [ S.Pop; S.Pop ]) ]
   else []
 
-let expr_available_2_f64_i32 (env : Env.t) =
+let expr_available_2_f64_i32 (env : Env.t) : _ condgen =
+  return
+  @@
   if B.memory_exists env then [ pair B.f64_store (const [ S.Pop; S.Pop ]) ]
   else []
 
-let expr_available_3_i32 env =
+let expr_available_3_i32 env : _ condgen =
   let mem_expr =
     if B.memory_exists env then
-      [ B.memory_copy; B.memory_fill ] @ B.memory_init env
+      [ return @@ B.memory_copy; not_symbolic @@ B.memory_fill ]
+      @ List.map (fun e -> return @@ e) (B.memory_init env)
```

Comment:
`@ List.map return (B.memory_init env)`

(Also, using `return @@ e` instead of `return e` is a little bit weird)

Reply:
An oversight during refactoring